Sources/Extensions/UIKit/UIDeviceExtensions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIDeviceExtensions.swift
+//  SwifterSwift
+//
+//  Created by Anton Novoselov on 31/12/2017.
+//  Copyright © 2017 SwifterSwift
+//
+
+#if os(iOS) || os(tvOS)
+import UIKit
+public extension UIDevice {
+    /// Show country code of current locale for device
+    static var countryISOCode: String? {
```

Comment:
Hey @n0an, @omaralbeik, @SD10 would it make more sense being an instance property? To me it make more sense to someone using the API use `UIDevice.current.countryISOCode`  than `UIDevice.countryISOCode` since I want the current device code. It seems more verbose but I think it's more readable and make more sense. Just a suggestion, What you guys think? :)

Reply:
I agree with @LucianoPAlmeida 👍 Welcome back!

Reply:
Welcome back @LucianoPAlmeida!
I agree with you about making it an instance property, but I still think that the right place for this extension is `Locale`

Reply:
@omaralbeik I agree, this makes more sense as a `Locale` extension :)

Reply:
I don't mind. Moved to Locale extensions.